packages/cli/bin/_update-template-deps.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+/**
+ * This is an internal utility to update `lib/dependencies.json`, which declares
+ * a set of dependencies used by our CLI templates for projects.
+ *
+ * Usage:
+ * ```
+ * node bin/_update-template-deps.js >lib/dependencies.json
```

Comment:
Can we make this a part of the `package.json` scripts for this package (`@loopback/cli`). Or better yet, just make it a part of the release process since that's when we'd run this anyways.

Reply:
I intentionally didn't automate it to avoid git commit of the updated `lib/dependencies.json` during the release process. The script only prints content to stdout to give us more control.

Reply:
Can we at least create an npm script that people can run manually? The current usage is too long to my taste. Ideally, I'd like to run something like the following:

```shell
# in monorepo root
$ lerna run update-template-deps
# in packages/cli
$ npm run update-template-deps
```